week-2/mandatory/2-function-creation.js

```diff
@@ -0,0 +1,145 @@
+/*
+Write a function that:
+- takes an array of strings as input
+- removes any spaces in the beginning or end of the strings
+- removes any forward slashes (/) in the strings
+- makes the string all lowercase
+*/
+// ["/Daniel ", "irina ", " Gordon", "ashleigh "]
+function tidyUpString(strArr) {
+    for (let i = 0; i < strArr.length; i++) {
+        strArr[i] = strArr[i].trim().toLowerCase()
+        strArr[i] = strArr[i].replace("/", "")
+    }
+    return strArr
```

Comment:
Nice! Good job! 

A little extra thing to consider here that is asked later down the file as well. Does `strArr` that is passed as input change in this case?  In case it does, how would you change the function so that it does not change the input? There were not requirements about it this time, but it might be an interesting thing to consider.